app/views/gobierto_admin/gobierto_people/configuration/settings/edit.html.erb

```diff
@@ -39,6 +39,17 @@
         <%= f.label :calendar_integration, t(".calendar_integration") %>
         <%= f.select :calendar_integration, @calendar_integrations_options, include_blank: true %>
       </div>
+
+      <div class="form_item input_text">
+        <%= f.label :ibm_notes_usr, t(".ibm_notes_usr") %>
+        <%= f.text_field :ibm_notes_usr, placeholder: t(".placeholders.ibm_notes_usr") %>
+      </div>
+
+      <div class="form_item input_text">
+        <%= f.label :ibm_notes_pwd, t(".ibm_notes_pwd") %>
+        <%= f.text_field :ibm_notes_pwd, placeholder: t(".placeholders.ibm_notes_pwd") %>
+      </div>
+
```

Comment:
Are you going to add the javascript code to clean the form? Is that why it's a WIP?